src/executors/IntervalExecutor.js

```diff
@@ -4,6 +4,19 @@ import {getArgNames} from '../util/function';
 import {buildArgumentsFromRingaEvent} from '../util/executors';
 
 class IntervalExecutor extends ExecutorAbstract {
+  //-----------------------------------
+  // Constructor
+  //-----------------------------------
+  /**
+   * Constructor for a new IntervalExecutor.
+   *
+   * @param thread The parent thread that owns this command.
+   * @param condition The interval keeps looping so long as this returns truthy
+   * @param executor  The executor that runs on every loop
+   * @param milliseconds The time between each loop
+   * @param options Defaults are provided, so this is optional.
+   * @param options.maxLoops Maximum number of loops before killing IntervalExecutor
```

Comment:
'Killing' implies thread kill to me. Let's be more explicit and say "When maximum loops has occurred, done() is called on the executor".

Also, if maximum loops is -1, then it should never stop. It should be -1 by default, in my opinion.